urllib3/response.py

```diff
@@ -351,3 +363,77 @@ def readinto(self, b):
         else:
             b[:len(temp)] = temp
             return len(temp)
+
+    def _read_chunked(self, amt):
```

Comment:
I took the prototype directly [from httplib](https://hg.python.org/cpython/file/2.7/Lib/httplib.py#l590). Will fix.